Infrastructure/Extensions/TuitionSettingExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Application.Extensions;
+using TuitionSetting = Domain.Enums.TuitionSetting;
+
+namespace Infrastructure.Extensions
+{
+    public static class TuitionSettingExtensions
+    {
+        public static string GetTuitionSettingName(this ICollection<Domain.TuitionSetting>? tuitionSettings)
+        {
+            if (tuitionSettings != null && tuitionSettings.Any())
+            {
+                if (tuitionSettings.Count() == 1)
```

Comment:
"IEnumerable" LINQs should be simplified. Use `Count` property here instead.

i.e: `tuitionSettings.Count == 1`